src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraClientImpl.java

```diff
@@ -140,13 +145,13 @@ public URL calculateSinglePaymentURL(String correlationId, BamboraTransType tran
 			redirect = this.hostedPaymentURL + "?" + paramString;
 	
 			 //TODO - Complete logging once available.
-			//logger.info("Single Payment URL calculated as: " + redirect);
+			logger.info("Single Payment URL calculated as: " + redirect);
```

Comment:
same here, do we need to push `redirect` to the log?

Reply:
No need to send this to the output. Again, was just for debugging purposes.

Reply:
removed from code